### DISCUSSION TITLE
Poste.io CapRover Let's Encrypt connection refused

### DISCUSSION THREAD

**Discussion by Toninobandolero** [caprover/caprover#1327] [Q&A]
Hello
I have poste.io running in the caprover environment. I think my problem has to do with the TLS Certificate. Since Caprover manages these certificates with Let's Encrypt.
What happens is that apparently everything works on post.io mail server. I can access the secure web provided by Caprover. **I can send and receive mail from webmail.**

When I try to configure mail in a client, such as macOS mail, the problem occurs. The IMAP server is working, but I can't connect to the SMTP server. I use the settings recommended by poste.io:
![pantallazo poste io](https://user-images.githubusercontent.com/63654670/152638961-14779ff5-2750-4ac1-a82a-a035c15883af.jpg)

Submission (SMTP for clients)
* host: mail.myweb.xyz
* port: 465
* **TLS with authentication required**
* username is whole email address username@domain.com

IMAP (preferred)
* host: mail.myweb.xyz
* port: 993
*TLS
* username is whole email address username@domain.com

or POP3
* host: mail.myweb.xyz
* port: 995
*TLS
* username is whole email address username@domain.com

I think this all has to do with the fact that I can't generate TLS from post.io

So is there an easy way to configure post.io so that it gets the certificate so I can configure my email client?

**Comment by githubsaturn** [comment]
This is a Poste.io questions. I suggest reaching out to their forum. With regards to CapRover, all certificates are automatically stores and renewed. They are located in 
```
/captain/data/letencrypt/etc/live
```

You just need to map it to where Poste.io is expecting these keys to be. You can ask Poste.io folks how to map these keys.

**Reply by Toninobandolero** [reply]
I have asked to the poste.io guys by email:

I managed to install it from the one-click apps option of caprover. everything has been installed and works correctly, except the TLS certificates, which, being managed by caprover, are not generated correctly from the poste.io mailserver settings page.

Apparently the problem could be solved by adding the path 

/captain/data/letencrypt/etc/live

where the caprover certificates are stored, to the path where poste.io would expect these certificates.

My question is: where should I access within poste.io to change the default route to this new route?

So let see what they answer

**Reply by b3nab** [reply]
Any updates? I also saw a tutorial on how to overcome ssl certificate issues https://github.com/caprover/one-click-apps/issues/231#issuecomment-813321994

**Comment by lgarcia-sfisas** [comment]
How did you do to enable Https, I try from Caprover but it just won't let me because of such an error:

`Unexpected output when enabling SSL formail.apps.luacgn.com with ACME Certbot Saving debug log to /var/log/letsencrypt/letsencrypt.log Plugins selected: Authenticator webroot, Installer None Obtaining a new certificate Performing the following challenges: http-01 challenge for mail.apps.luacgn.com Using the webroot path /captain-webroot/mail.******.luacgn.com for all unmatched domains. Waiting for verification... [31mChallenge failed for domain mail.****.luacgn.com[0m Challenge http-01 for mail.apps.*****.com Clearing challenges [31mSome challenges have failed.[0m [1m IMPORTANT NOTES: [0m - The following errors were reported by the server: Domain: mail.apps.luacgn.com Type: unauthorized Detail: 159.***.205. *** : Invalid response from http://mail.apps.*******.com/.well-known/acme-challenge/HFSn8qH7o86XvElc1zs8JaCdSDA6h3G21anif9PyYUc: 404 To fix these errors, make sure that your domain name is entered correctly and that the DNS A/AAAA records for that domain contain the correct IP address.`

**Comment by locvfx** [comment]
https://github.com/caprover/one-click-apps/issues/231
